custom/conf/app.example.ini

```diff
@@ -2125,6 +2125,8 @@ PATH =
 ;RENDER_COMMAND = "asciidoc --out-file=- -"
 ;; Don't pass the file on STDIN, pass the filename as argument instead.
 ;IS_INPUT_FILE = false
+; Don't filter html tags and attributes if true
+;DISABLE_SANITIZE = false
```

Comment:
I think we should use `Sanitizer` instead of `Sanitize` (setting key/variable name) in most cases. `Sanitize` is a verb, and there were names like `SanitizerRules` uses `Sanitizer`, using Sanitizer is more consistent.